lib/api/v3/projects/project_sql_representer.rb

```diff
@@ -52,7 +57,6 @@ def ancestors_sql(walker_result)
                   FROM projects origin
                   LEFT OUTER JOIN projects ancestors
                   ON ancestors.lft < origin.lft AND ancestors.rgt > origin.rgt
-                  #{visibility_join}
```

Comment:
Can `visibility_join` method be removed now?

Reply:
Of course, thanks for spotting, @cbliard